pyrefly/lib/alt/call.rs

```diff
@@ -1,14 +1,15 @@
 /*
- * Copyright (c) Meta Platforms, Inc. and affiliates.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- */
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
For the most part the change makes sense, not sure what happened here, lots of lines with no obvious change are highlighted.

Maybe a carriage return issue? Just a wild guess

At any rate, if you're making a new branch we should just keep an eye on this